src/ModelRouter.ts

```diff
@@ -76,9 +76,11 @@ export class RouterModes {
  * @return
  */
 export function getModelPath(url?: string | null | URL): string | null {
-    const localUrl = (url || window.location.pathname) as string;
-
-    return PathUtils.sanitize(localUrl);
+    const localUrl = (url || (PathUtils.isBrowser() && window.location.pathname)) as string;
+    if (localUrl) {
+        return PathUtils.sanitize(localUrl);
```

Comment:
Please fix the linting errors in the changes.

Reply:
Fixed in 1295e88201256be8e5c27a23f0f5ff0143c14b95.